demos/bridge/primefaces3-portlet/src/main/webapp/views/applicant.xhtml

```diff
@@ -115,7 +113,7 @@
 		<alloy:column id="c2" width="33">
 			<alloy:row>
 				<alloy:column id="c2a" styleClass="uploaded-files">
-					<h:form id="f1" enctype="multipart/form-data" prependId="false">
+					<h:form id="f2" enctype="multipart/form-data" prependId="false">
```

Comment:
@stiemannkj1 I noticed that this line was changed from "f1" to "f2" in the primefaces3-portlet. Does it need to get fixed in the others too or are they OK?